### DISCUSSION TITLE
Get clicked marker info

### DISCUSSION THREAD

**Discussion by tk-dg** [giswqs/streamlit-geospatial#135] [Q&A] [answered]
Hi there,

Is there a way to identify which marker was clicked on this page without using JavaScript? I understand that streamlit_folium offers two-way binding, but due to the slow loading times with folium, I would prefer a solution that involves only Leaflet and Python, if possible.

Here is the reference link for context: https://github.com/opengeos/streamlit-geospatial/blob/master/pages/5_%F0%9F%93%8D_Marker_Cluster.py

Thank you!

**Comment by giswqs** [comment, marked as answer]
Use solara,which has native support for ipyleaflet and ipywidgets

https://github.com/opengeos/solara-geospatial
